web/pdf_page_view.js

```diff
@@ -1001,6 +1026,14 @@ class PDFPageView {
           await this.#renderAnnotationLayer();
         }
 
+        if (!this.drawLayer) {
+          this.drawLayer = new DrawLayerBuilder({
+            pageIndex: this.id,
+          });
+        }
+        await this.#renderDrawLayer();
+        this.drawLayer.drawLayer.setParent(canvasWrapper);
+
```

Comment:
This is only necessary when editing is supported, right?
Because if so this shouldn't be done *unconditionally*, and this entire block should be wrapped in an `if (annotationEditorUIManager) { ... }` block. (Assuming that we move [this line](https://github.com/mozilla/pdf.js/blob/f4b396f6c88e951174879a52390fa89223326b36/web/pdf_page_view.js#L1005) up to this level of the code.)